src/index.html

```diff
@@ -1009,6 +1009,23 @@ <h2 class="app__title">
 			</style>
 		</a>
 
+		<a class="list__item app js-app"
+				href="https://resilientwebdesign.com/"
+				data-app="ResilientWebDesign"
+				data-tags="other">
+			<div class="app__wrapper">
+				<h2 class="app__title">
+					RESILIENT WEB DESIGN
```

Comment:
This could just say “Resilient Web Design”, in title case. (CODE.NASA.GOV uses all caps because that’s the way the name is stylized.)